docs/docs-ref-autogen/excel/excel/excel.interfaces.slicercollectionloadoptions.yml

```diff
@@ -173,8 +173,8 @@ properties:
     package: excel!
     fullName: sortBy
     summary: >-
-      For EACH ITEM in the collection: Represents the sort order of the items in
-      the slicer. Possible values are: "DataSourceOrder", "Ascending",
+      For EACH ITEM in the collection: Specifies the sort order of the items in
+      the slicer. Possible values areString: "DataSourceOrder", "Ascending",
```

Comment:
This seems like a processor error. I don't see "areString" anywhere in DefinitelyTyped.

Reply:
taking a look...

Reply:
https://github.com/OfficeDev/office-js-docs-reference/pull/2313 Should fix this issue.